look-and-feel/react/src/List/ClickList/ClickItem/types.d.ts

```diff
@@ -0,0 +1,22 @@
+import type { ComponentProps, ReactNode } from "react";
+
+export type TClickItem = {
+  classModifier?: string;
+  isDisabled?: boolean;
+} & Omit<ComponentProps<"button">, "disabled"> &
+  ComponentProps<"a"> & {
+    /**
+     * @deprecated Use children prop instead
+     */
```

Comment:
**question**: toujours d'actualité ?

Reply:
En soit oui vu qu'il c'est mieux d'utiliser `children` que `label`.
Tu penses que ce serait préférable de juste supprimer la propriété `label` vu qu'on est pas sur une version stable ?

Reply:
Pour moi c'était plutôt un oubli
Je pense qu'on peut la supprimer oui

Reply:
Fixed in : https://github.com/AxaFrance/design-system/pull/593/commits/ab8d2232d723ad9c2099450a02c9d963a848527f